src/main/java/frc/robot/Constants.java

```diff
@@ -32,9 +34,27 @@ public final class Constants {
       Seconds.of(0.13); // s, 20ms + 110ms sprk max velocity lag
   public static final Measure<Velocity<Distance>> MAX_SPEED = FeetPerSecond.of(14.5);
 
-  public static final class AutoConstants {
+  public static final class PivotConstants {
+    public static final double VORTEX_ENCODER_TICKS_PER_REVOLUTION = 7168;
     public static final PIDConstants TRANSLATION_PID = new PIDConstants(0.7, 0, 0);
     public static final PIDConstants ANGLE_PID = new PIDConstants(0.4, 0, 0.01);
+    public static final double PIVOT_kP = 0;
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0;
+    public static final double PIVOT_kS = 0;
+    public static final double PIVOT_kG = 0;
+    public static final double PIVOT_kV = 0;
+    public static final double PIVOT_kA = 0;
+    public static final int PIVOT_MOTOR = 20;
+    public static final Measure<Velocity<Angle>> PIVOT_MAX_VEL = RPM.of(4);
```

Comment:
So this is actually the same unit as whatever we do for getMeasurement() in the PivotSubsystem.

Reply:
See here it's rads per second: https://docs.wpilib.org/en/stable/docs/software/commandbased/profile-subsystems-commands.html

Reply:
And here: https://docs.wpilib.org/en/stable/docs/software/commandbased/profilepid-subsystems-commands.html